spec/inputs/http_poller_spec.rb

```diff
@@ -219,7 +219,7 @@
 
       before do
         Timecop.travel(Time.new(2000,1,1,0,0,0,'+00:00'))
-        Timecop.scale (60 * 5) / 2
+        Timecop.scale (60)
```

Comment:
Sorry if I missed it, but what are the adjustments to the schedule tests? Is t to reduce test time/resource utilization? A race condition?

Reply:
I've reworked this slightly to be less of a change, and will include the following comment in the squashed merge:

```
 This commit also fixes a failing test, where an aggressive Timecop configuration
    meant that the setup for the plugin was not complete by the time the expected
    scheduled job was to be run
```